### PR TITLE
Start dcos package in Go.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

File: pkg/packages/client.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package packages
```
Comment (rgo3) [inline pkg/packages/client.go]:
```suggestion
package cosmos
```

Comment (rgo3) [reply pkg/packages/client.go]: Or is there a reason to not name it `cosmos` ?

File: pkg/cmd/pkg/package_install.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+func newCmdPackageInstall(ctx api.Context) *cobra.Command {
+	var appOnly, cliOnly, global, yes bool
+	var appID, optionsPath, version string
+
+	cmd := &cobra.Command{
+		Use:   "install <package-name>",
+		Short: "Install a package",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return nil
```
Comment (rgo3) [inline pkg/cmd/pkg/package_install.go]: Probably we also want to invoke the Python CLI here for now?

File: pkg/cmd/pkg/package_repo_add.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+	cmd := &cobra.Command{
+		Use:   "add <repo-name> <repo-url>",
+		Short: "Add a package repository to DC/OS",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return invokePythonCLI(ctx)
+		},
+	}
+	cmd.Flags().IntVar(&index, "index", 0, "Nmerical position in the package repository list. Package repositories are searched in descending order.")
```
Comment (rgo3) [inline pkg/cmd/pkg/package_repo_add.go]:
```suggestion
	cmd.Flags().IntVar(&index, "index", 0, "Numerical position in the package repository list. Package repositories are searched in descending order.")
```

**Comment by ArmandGrillet** [comment]
@mesosphere-ci run integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Review by bamarni — COMMENTED** [review]
LGTMWHENCIPASSES

**Comment by armandgrillet** [comment]
@mesosphere-ci run integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Review by rgo3 — APPROVED** [review]

**Review by bamarni — APPROVED** [review]

**Pull request merged**